pandas/core/indexes/category.py

```diff
@@ -147,6 +158,20 @@ class CategoricalIndex(Index, accessor.PandasDelegate):
 
     _typ = "categoricalindex"
 
+    _raw_inherit = {
```

Comment:
i find this name hard to grok, these are the delegates to the array right, so can we call this
_delegated_to_array?

Reply:
yah, im planning on doing a refactor to clean all of these up